actions/contributing-update/README.md

```diff
@@ -1,7 +1,24 @@
-### Contributing Workflow
+## Contributing Workflow
 
 This action checks the contributing file for a charm and updates it if necessary. The action runs a script that generates a fresh contributing file from a template and placeholder values defined at the charm level. This generated file is compared against the contributing file already in the charm's directory. If they differ, another step is run to create a PR overwriting the existing file with the freshly generated one.
 
-### contributing_inputs.yaml
+## contributing_inputs.yaml
 
-This file, existing in charm's directory, defines all charm-specific inputs that will be injected into the `contributing.md.template` file to form the new `contributing.md` file. If this file doesn't exist, or doesn't contain definitions for all template values, the contributing update will fail, as there won't be enough information to generate the new `contributing.md` file from the template.
\ No newline at end of file
+This file, existing in charm's directory, defines all charm-specific inputs that will be injected into the `contributing.md.template` file to form the new `contributing.md` file. If this file doesn't exist, or doesn't contain definitions for all template values, the contributing update will fail, as there won't be enough information to generate the new `contributing.md` file from the template.
+
+## Updates
+
+### Charm Specific
+
+Whenever we want to make changes to a charm contributing.md, we:
+
+1. Update `contributing_inputs.yaml`
+1. We can also update the `contributing.md` file directly if we like, if we want to avoid another PR
```

Comment:
Do you think it'd make more sense to discourage people from editing it directly? Sort of defeats the purpose of this automation IMO.

Reply:
Yes. I do. I think this is a hacky use case as you mention in your other comment. I think the follow-on issue will address that, and the README will need to be updated when it's done.

For now though, I think it's best to document the action as it is today, seeing as it's already merged and this is how it actually works.